src/utils/deepLinks.ts

```diff
@@ -1,7 +1,7 @@
 import {PROJECT_NAME_PLURAL} from 'src/flows'
 
 export const buildDeepLinkingMap = (orgId: string) => ({
-  '/me/about': `/orgs/${orgId}/about`,
+  '/me/org-settings': `/orgs/${orgId}/org-settings`,
```

Comment:
let's keep the `/me/about` and `/me/users` deeplinks around for backwards link compatibility, but have them link to the new link. Then let's remove them when the old routes are removed from `SetOrg.tsx`

```ts
'/me/about': `/orgs/${orgId}/org-settings`
...
'/me/users': `/orgs/${orgId}/members`
```